pdc/apps/common/renderers.py

```diff
@@ -142,11 +144,9 @@ def format_docstring(self, view, method, docstring):
         macros = settings.BROWSABLE_DOCUMENT_MACROS
         if view:
             macros['FILTERS'] = get_filters(view)
-            if 'list' == method:
-                ordering_field = get_ordering_field(view, method)
-                if ordering_field:
-                    ordering_string = ORDERING_STRING + " %s . " % ordering_field + ORDERING_STRING_NESTED
-                    macros['FILTERS'] += ordering_string
```

Comment:
That make sense to me.
One question, can you add the jira id in commit log?
Another question, do you mean the original codes in ordering is like this:
When we want to ordering by arch, we need to use variant_arch__arch__name, but with this patch we just need use "arch" ? 

If so, did you forget to remove some codes?
like remove code in _check_ordering_keys function? 
```
        invalid_fields = set(tmp_list) - set(valid_fields)
        if invalid_fields:
            raise FieldError('Unknown query key: %s not in fields: %s' %
                             (list(invalid_fields), valid_fields))
```

I think this patch is pretty cool, but need to remove these codes first.

Reply:
> One question, can you add the jira id in commit log?

JIRA is already mentioned (github doesn't update PR description after amending/adding commits; you need to go to "Commits" tab here and click on the commit).

> If so, did you forget to remove some codes?
> like remove code in _check_ordering_keys function

I think the function work fine even with the change, but I'll double-check that.

Reply:
OK, thanks for reminding me that.

Reply:
OK, should be fixed.

BTW, maybe the ` _check_ordering_keys` could be removed  later (~25 lines of code in total). The only difference when not using it is that the list of valid ordering fields is not printed in an exception on server side. Also this is probably **a bug** because the error should be passed to client instead.

Reply:
Yes, I think so, maybe remove from 198 to 213 lines of common.viewsets.py

Reply:
@xiangge It would be better to change it in another patch (new JIRA?). The error should be passed to client (JSON `{"detail":"..."}` instead of HTML error), not sure if this will be possible if the code is removed.

Reply:
@hluk seems that, if you don't move the four lines I mentioned, when you use the arch to order, on content-delivery-repo, it can't request the data and will raise error.
Here is my local test, just open this below url in browser, not use `curl -d`
http://10.66.128.226:8088/rest_api/v1/content-delivery-repos/?ordering=arch
Then we can see the error:
```
FieldError at /rest_api/v1/content-delivery-repos/
Unknown query key: [u'arch'] not in fields: ['name', 'service', 'content_format', 'content_category', 'shadow', 'repo_family', u'id', 'product_id']

```
So I think we can remove several lines first in this patch.
And then create issue about removing this function. What is your thinking?

Reply:
I already patched the function - I don't see any server exception on localhost.

I'll create new issue for the server exception.

Reply:
Cool, then, this looks good for me. 
And I think that patch and this patch should in the same release.